tests/unit/validation/test_batch_validation.py

```diff
@@ -53,6 +54,16 @@ def batch_data() -> dict[str, np.ndarray]:
     return {"line": line, "asym_load": asym_load}
 
 
+@pytest.fixture
+def original_batch_data_columnar(original_batch_data):
+    return compatibility_convert_row_columnar_dataset(original_batch_data, ..., DatasetType.update)
```

Comment:
it's a nitpick, but `...` looks weird here. Maybe it's a good idea to be explicit.

```suggestion
    return compatibility_convert_row_columnar_dataset(original_batch_data, Ellipsis, DatasetType.update)
```

it's equivalent, but more clearly tells the reader that it's a deliberate special value that is different than just "everything"/"et cetera" (which is what the ellipsis stands for)

the way i read the current state is also:
* "return (for compatibility) a converted row/columnar data set from the original batch data etc. as an update data set type".

as opposed to
* "return (for compatibility) a converted row/columnar data set from the original batch, using ellipsis and update data set type as parameters"

alternatively, you can also use explicit keyword arguments